environment-dev.yml

```diff
@@ -6,6 +6,8 @@ dependencies:
   # Don't forget to sync changes between environment.yml, environment-dev.yml, and pyproject.toml!
   # Main packages
   - numpy
+  - pooch >=1.8.0
+  - pydantic >=2.0,<2.5.3
```

Comment:
```suggestion
  - pydantic >=2.0,<2.5.3  # FIXME: Remove pin once our dependencies (xclim, xscen) support pydantic 2.5.3
```

Reply:
I'm not sure if the pin is supposed to be permanent or temporary. If temporary, I'd add this.